portal-frontend/src/app/features/edit-application/edit-application.component.ts

```diff
@@ -79,6 +79,9 @@ export class EditApplicationComponent implements OnInit, OnDestroy, AfterViewIni
           this.loadApplication(fileId).then(() => {
             const stepInd = paramMap.get('stepInd');
             const parcelUuid = queryParamMap.get('parcelUuid');
+            if (queryParamMap.get('errors')) {
```

Comment:
just a nitpick
const isShowError = queryParamMap.get('errors')
if(isShowError) {
this.showValidationErrors = isShowError ==='t'
}

Reply:
Done!